automation_tools/__init__.py

```diff
@@ -444,6 +444,42 @@ def setup_default_libvirt(bridge=None, ip_address="192.168.100.1"):
     return interface
 
 
+def setup_http_proxy():
+    """Postinstall task to set http_proxy and content_default_http_proxy settings for
+    Satellite >= 6.7
+
+    Proxy information is passed using the PROXY_INFO environmental variable.
+    The expected format is::
+
+        PROXY_INFO=proxy://<username>:<password>@<hostname>:<port>
+
+    ``username`` and ``password`` fields can be omitted if the proxy does not
+    require authentication, for example::
+
+        PROXY_INFO=proxy://<hostname>:<port>
+    """
+    proxy_info = os.environ.get('PROXY_INFO')
+    if proxy_info is None:
+        print('The PROXY_INFO environment variable should be defined')
+        sys.exit(1)
```

Comment:
This function is gated behind `if os.environ.get('PROXY_INFO')`. I wonder in what scenario we can call it and then have variable undefined under our feet.

Reply:
@mirekdlugosz Updated the  PR.